pkg/controller/operands/cdi_test.go

```diff
@@ -402,6 +413,183 @@ var _ = Describe("CDI Operand", func() {
 				Message: "CDI is degraded: Bar",
 			}))
 		})
+
+		Context("Unsupported Annotation", func() {
```

Comment:
I think the term "unsupported annotation" is now left at design doc/JIRA issues only, and throughout the code is either referred to as "tainted configuration" or "jsonpatch annotation". Consider using one of these two, to emphasize that this relates to the same feature.

Reply:
These are unit test. One checks the `NewCDI` func and one checks the `ensure` func. Not sure I get the comment. We need to make sure that NewCDI works *and* that the `ensure` uses it corrctlly.